core/core/models/payload.py

```diff
@@ -14,8 +14,12 @@ class ContactPayload(BaseModel):
 class BirthdayPayload(BaseModel):
     name: str
     birthday: Optional[str] = ''
-
-
+   
+    
+class AllBirthdaysPayload(BaseModel):
+    day_amount: Optional[str] = '7'
```

Comment:
why string?

Reply:
It wasn't a problem in the futurefor us to use an int? If no, sure I edit it

Reply:
Done

Reply:
Yes, it should be okay. We can use any data type we want.